src/commands/pulls/pulls.ts

```diff
@@ -125,7 +125,16 @@ export async function execute(interaction: CommandInteraction, client: Client) {
         components: [row],
       })
 
-      await pinMessage(thread)
+      const messages = await thread.messages.fetchPinned()
+
+      const messageInfo = messages.at(0)
+      console.log(messageInfo)
```

Comment:
e esse console aqui

Reply:
Acredito que nao tem como ficar undefined nao, só se for em thread MUUUITO antiga, a maioria já morreu.